doc/lightning-fundchannel.7.txt

```diff
@@ -0,0 +1,54 @@
+LIGHTNING-FUNDCHANNEL(7)
+========================
+:doctype: manpage
+
+NAME
+----
+lightning-fundchannel - Command for establishing a lightning channel.
+
+SYNOPSIS
+--------
+*fundchannel* 'id' 'satoshi'
+
+DESCRIPTION
+-----------
+The *fundchannel* RPC command opens a payment channel with a peer by commiting
+a funding transaction to the blockchain as defined in BOLT #2.  Once the
+transaction is confirmed, normal channel operations may begin.  Readiness is
+indicated by *listfunds* reporting a 'short_channel_id' for the channel.
+Channel status can then be monitored using *listchannels*.
```

Comment:
`listchannels`, despite its name, lists all channels known in the public network, i.e. the entire routemap.  Local channels are listed in `listpeers`.